src/frontend/components/Felleskomponenter/Barnetrygdperiode/useBarnetrygdperiodeSkjema.tsx

```diff
@@ -0,0 +1,68 @@
+import { ESvar } from '@navikt/familie-form-elements';
+import { useSkjema, Valideringsstatus } from '@navikt/familie-skjema';
+
+import useDatovelgerFelt from '../../../hooks/useDatovelgerFelt';
+import useInputFelt from '../../../hooks/useInputFelt';
+import useJaNeiSpmFelt from '../../../hooks/useJaNeiSpmFelt';
+import useLanddropdownFelt from '../../../hooks/useLanddropdownFelt';
+import { IBarnetrygdperioderFeltTyper } from '../../../typer/skjema';
+import { dagensDato, gårsdagensDato } from '../../../utils/dato';
+import { barnetrygdslandFeilmelding } from './barnetrygdperiodeSpråkUtils';
+import { BarnetrygdperiodeSpørsmålsId } from './spørsmål';
+
+export const useBarnetrygdperiodeSkjema = () => {
+    const mottarEøsBarnetrygdNå = useJaNeiSpmFelt({
+        søknadsfelt: { id: BarnetrygdperiodeSpørsmålsId.mottarBarnetrygdNå, svar: null },
+        feilmeldingSpråkId: 'modal.barnetrygdnå.feilmelding',
+    });
+
+    const tilbakeITid = mottarEøsBarnetrygdNå.verdi === ESvar.JA;
+
+    const barnetrygdsland = useLanddropdownFelt({
+        søknadsfelt: { id: BarnetrygdperiodeSpørsmålsId.barnetrygdsland, svar: '' },
+        feilmeldingSpråkId: barnetrygdslandFeilmelding(tilbakeITid),
+        skalFeltetVises: mottarEøsBarnetrygdNå.valideringsstatus === Valideringsstatus.OK,
+        nullstillVedAvhengighetEndring: true,
+    });
+
+    const fraDatoBarnetrygdperiode = useDatovelgerFelt({
+        søknadsfelt: { id: BarnetrygdperiodeSpørsmålsId.fraDatoBarnetrygdperiode, svar: '' },
+        skalFeltetVises: mottarEøsBarnetrygdNå.valideringsstatus === Valideringsstatus.OK,
+        feilmeldingSpråkId: 'modal.trygdnårbegynte.feilmelding',
+        sluttdatoAvgrensning: tilbakeITid ? gårsdagensDato() : dagensDato(),
+        nullstillVedAvhengighetEndring: true,
+    });
+
+    const tilDatoBarnetrygdperiode = useDatovelgerFelt({
+        søknadsfelt: { id: BarnetrygdperiodeSpørsmålsId.tilDatoBarnetrygdperiode, svar: '' },
+        skalFeltetVises:
+            mottarEøsBarnetrygdNå.valideringsstatus === Valideringsstatus.OK &&
+            mottarEøsBarnetrygdNå.verdi === ESvar.NEI,
```

Comment:
Disse to er overlappende, fordi hvis verdien er ESvar.Nei er også validersstatusen OK 😊

Reply:
Her kan bruke den tilbakeITid variabelen og

Reply:
Tar i bruk tilbakeITid 😊